x-pack/platform/plugins/shared/cases/common/types/domain/suggestion/v1.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { z } from '@kbn/zod';
+import type { CaseAttachmentWithoutOwner } from '../attachment/v1';
+import { OWNERS } from '../../../constants/owners';
+
+const suggestionOwnerSchema = z.enum(OWNERS);
+
+export const suggestionContextRt = z.object({
+  'service.name': z.string().optional(),
+  timeRange: z
+    .object({
+      from: z.string(),
+      to: z.string(),
+    })
+    .optional(),
+});
```

Comment:
I'd keep it simple for now, we can add new properties when needed.
Also changed `service.name` to be an array of strings, that is what it will be returned by the `getMetadata` of the cases client.

```suggestion
export const suggestionContextRt = z.object({
  'service.name': z.array(z.string()).optional(),
});
```

Reply:
I'm quite certain @fkanout will immediately need this for querying the synthetics indices.